### ISSUE TITLE
who at discord is responsible for this? i need names.

### ISSUE THREAD

**Issue by mr-miner1** [mr-miner1/Better-Badges#6]
discord thought it was a great idea to fuck up capitalization of aria-labels very cool, imma wait and see if its a bug from discord's side and if they fix it and im planning to change the fetching of badges from aria-label's to src anyway cause language support and blah blah should be fixed in less than a week (long deadline cause i have exams currently)
![image](https://user-images.githubusercontent.com/83544549/126807033-1ae8ec49-5beb-4bb5-b231-6a9b8fc61875.png)

**Comment by spinfish** [comment]
I could try to do if you wanted me to?

**Comment by mr-miner1** [comment]
> I could try to do if you wanted me to?
nah its cool im basically done with the "colored badges" just need to fix the white ones should be done by today
> ~And yeah whoever is working as a dev at Discord I swear to God must be on crack~
fr

**Comment by spinfish** [comment]
Alright, epico B)

**Comment by mr-miner1** [comment]
god damn it, wasnt a discord update but me changing my language from eng us to eng uk lmao (still weird why is the uk one not capitalized) 
anyway rewrote the theme to use svgs now apart from early supporter(dont think i can do anything about this) and booster badges(if someone wants to get all the 11 svgs and fix this be my guest, make a pull request or smth)
